tests/client.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
```

Comment:
To avoid relative import, I think `client` can become a class into `conftest.py` available as a fixture

Reply:
I see your point. I can live with that for now though ;-)